webpack/scenes/Subscriptions/Details/SubscriptionDetails.js

```diff
@@ -1,33 +1,69 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Grid, Row, Col } from 'patternfly-react';
+import BreadcrumbsBar from 'foremanReact/components/BreadcrumbBar';
 import SubscriptionDetailInfo from './SubscriptionDetailInfo';
 import SubscriptionDetailAssociations from './SubscriptionDetailAssociations';
 import SubscriptionDetailProducts from './SubscriptionDetailProducts';
 import { LoadingState } from '../../../move_to_pf/LoadingState';
 import { notify } from '../../../move_to_foreman/foreman_toast_notifications';
 
 class SubscriptionDetails extends Component {
+  constructor() {
+    super();
+    this.handleBreadcrumbSwitcherItem = this.handleBreadcrumbSwitcherItem.bind(this);
+  }
   componentDidMount() {
     // eslint-disable-next-line react/prop-types
     const routerParams = this.props.match.params;
     this.props.loadSubscriptionDetails(parseInt(routerParams.id, 10));
   }
 
+  componentDidUpdate(prevProps) {
+    const routerParams = this.props.match.params;
+    if (routerParams.id !== prevProps.match.params.id) {
+      this.props.loadSubscriptionDetails(parseInt(routerParams.id, 10));
+    }
+  }
+
+  handleBreadcrumbSwitcherItem(e, url) {
+    this.props.history.push(url);
+    e.preventDefault();
+  }
+
   render() {
     const { subscriptionDetails } = this.props;
+    const resource = {
+      nameField: 'name',
+      resourceUrl: '/katello/api/subscriptions',
```

Comment:
Wonder if we should use [this](https://github.com/Katello/katello/blob/master/webpack/services/api/index.js#L27-L29) here or otherwise not hardcode this URL.